### ISSUE TITLE
Small disconnect between listed and actual lossy compression

### ISSUE THREAD

**Issue by LancetheGallade1500** [ImageOptim/ImageOptim#316]
I run Catalina 10.15.4.

The % of quality for Lossy Compression selected in Preferences differs from the value displayed in ImageOptim's interface.

I tried adjusting the .jpg percent for the first time to see if it would change the amount of optimization at all. But when I moved it back to 85%, the interface didn't display that number:

<img width="775" alt="Screen Shot 2020-06-15 at 12 10 35 PM" src="https://user-images.githubusercontent.com/48635148/84696725-c4372d80-af01-11ea-9b8b-a8264504fe8b.png">
<img width="661" alt="Screen Shot 2020-06-15 at 12 10 16 PM" src="https://user-images.githubusercontent.com/48635148/84696764-d2854980-af01-11ea-927f-6a84312592a0.png">

For some reason, it says 84% instead of 85%. I tried looking at the preferences file, and it said that JpegOptimMaxQuality was 84.70833333333333 instead of 85, like I would expect. I even tried manually changing it to 85 via Xcode, but when I opened ImageOptim back up and turned on Lossy Compression again, it still said "Lossy minification enabled (JPEG 84%)".

Then I tried moving the slider up and down to see if this issue occurs with any other percent. This issue starts appearing when you change the slider to 75% (it displays 74% on the bottom bar), and continues being 1% off until 99%, in which matches up again. In other words, the numbers only match up at 99% and at 50-72%.

So yeah. This is a _very_ minor issue, and the percents probably don't cause any real difference in the images produced. But it does bother my OCD a little that I can't set the percent back to where it was originally.

**Comment by kornelski** [comment]
Yeah, there are two different mechanisms involved in rounding these numbers.

**Comment by LancetheGallade1500** [comment]
I assumed it had something to do with rounding.

If you don’t mind me asking, why exactly is the rounding method different for those two?
